client/signup/index.web.js

```diff
@@ -20,6 +20,7 @@ export default function () {
 			`/start/:flowName/:stepName/${ lang }`,
 			`/start/:flowName/:stepName/:stepSectionName/${ lang }`,
 		],
+		controller.redirectTests,
```

Comment:
We'd want to redirect _only_ when they land on `/start/${ lang }`, right?

Reply:
In fact, we don't want any of the language URLs either unless they're `/en`.

Reply:
english-speaking users don't usually get their language added to the URL, most of the landing pages buttons redirects you just to /start

Reply:
>nglish-speaking users don't usually get their language added to the URL,

Yeah, that's why I thought it would be safe to just capture `/start` and not e.g. `/start/es`.

>most of the landing pages buttons redirects you just to /start

Nope, _all_ CTAs on localized wp.com homepage have the language and all the plans grid CTAs have also the bundle slug.